sympy/polys/tests/test_numberfields.py

```diff
@@ -637,9 +637,9 @@ def test_AlgebraicNumber():
 
     a = AlgebraicNumber(sqrt(2))
     b = to_number_field(sqrt(2))
-    assert a.args == b.args == (sqrt(2), Tuple())
+    assert a.args == b.args == (sqrt(2), Tuple(1, 0))
```

Comment:
Yes I mean `None` as default.  You know better than me how tightly coupled these changes are.

I'd like to ee the default changed to either `None` or `(1,0)` at the same time as your saved `scoeff` change that's all.  It also makes the changed test less controversial.  And you could maybe add `Tuple()` and `(0,0)` giving zero element tests too...